src/api/entities/Instruction/__tests__/index.ts

```diff
@@ -736,7 +736,7 @@ describe('Instruction class', () => {
         )
         .mockResolvedValue(expectedTransaction);
 
-      const tx = await instruction.reject();
+      const tx = await instruction.reject({});
```

Comment:
This is a breaking change. Is there any way to avoid requiring an explicit empty arg?

Reply:
As of now, the way `createProcedureMethod` is designed, we can't support that. https://github.com/PolymeshAssociation/polymesh-sdk/blob/alpha/src/utils/internal.ts#L641. Similar type of change is supposed to go for [this PR](https://github.com/PolymeshAssociation/polymesh-sdk/pull/1004) as well.